arctic/serialization/numpy_records.py

```diff
@@ -227,10 +227,32 @@ def _column_data(self, s):
         column_vals = [s.values]
         return columns, column_vals, None
 
-    def deserialize(self, item, _force_bytes_to_unicode=False):
+    def deserialize(self, item, force_bytes_to_unicode=False):
         index = self._index_from_records(item)
         name = item.dtype.names[-1]
-        return Series.from_array(item[name], index=index, name=name)
+        data = item[name]
+
+        if force_bytes_to_unicode:
```

Comment:
I looked at re-using the conversion code for the `DataFrame` deserialiser, but they ended up being subtly different.